README.md

```diff
@@ -185,28 +185,28 @@ ResourceSet rs = mc.resourceSet("mxwodetail").select("wonum","status").where((ne
 ResourceSet rs = mc.resourceSet("mxwodetail").select("wonum","status").where((new QueryWhere()).where("status").equalTo("APPR")).stablePaging(true).fetch();
 ```
 
-* Turn to next or previous page.
+* Turn to next or previous page:
 
 ```java 
 rs.nextPage();
 rs.previousPage();
 ```
 
-For stable paging where currently only scrolling forward is supported, a call to previousPage() would result in an error.
+For stable paging where currently only scrolling forward is supported, a call to previousPage() results in an error.
```

Comment:
195
For clarity, it would help to have an article and identifying noun here. 
The previousPage method?